platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/TaskInCalendar.java

```diff
@@ -0,0 +1,93 @@
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceHomePage;
+import org.exoplatform.platform.qa.ui.selenium.platform.social.SpaceManagement;
+import org.exoplatform.platform.qa.ui.task.pageobject.ProjectsManagement;
+import org.exoplatform.platform.qa.ui.task.pageobject.TasksManagement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selectors.byText;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.taskmanagement.TaskManagementLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+/**
+ * Created by esmaoui on 02/03/2018.
+ */
+@Tag("sniff")
+@Tag("task")
+public class TaskInCalendar extends Base {
+
+    HomePagePlatform homePagePlatform;
+
+    SpaceManagement  spaceManagement;
+
+    SpaceHomePage    spaceHomePage;
+
+    TasksManagement  tasksManagement;
+    TaskManagement   taskManagement;
+
+    ProjectsManagement projectsManagement;
+    CalendarManagement calendarManagement;
+    CalendarHomePage   calendarHomePage;
+
+
+    ManageLogInOut manageLogInOut;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+
+        homePagePlatform = new HomePagePlatform(this);
+        spaceHomePage = new SpaceHomePage(this);
+        spaceManagement = new SpaceManagement(this);
+        tasksManagement = new TasksManagement(this);
+        projectsManagement= new ProjectsManagement(this);
+        calendarManagement = new CalendarManagement(this);
+        calendarHomePage = new CalendarHomePage(this);
+        manageLogInOut = new ManageLogInOut(this);
+        manageLogInOut.signInCas(PLFData.USER_ROOT, PLFData.password);
+
+    }
+
+    @Test
+    @Tag("TA-605")
+    public void test01_AddTaskInSpace() {
+
+        String space = "space" + getRandomNumber();
+        String taskName = "task" + getRandomNumber();
+        String ContentTask = "contentTask" + getRandomNumber();
+
+
+        info("Create a space");
+        homePagePlatform.goToAllSpace();
+        spaceManagement.addNewSpaceSimple(space, space, 6000);
+        spaceManagement.goToTaskTab();
+        ELEMENT_LEFT_PANEL_IN_TASK_PAGE.find(byText(space)).click();
+        tasksManagement.addTask(taskName);
+        taskManagement.inputDataTaskInQuickForm(taskName,
+                ContentTask ,
+                getDate(0, "MM/dd/yyyy"),
+                getDate(0, "MM/dd/yyyy"),
+                false);
+        taskManagement.inputFromToQuickTask(null, null, true);
+        taskManagement.saveQuickAddTask();
+        projectsManagement.editProject(space, space, space, "", true);
+        homePagePlatform.goToCalendarPage();
+        ELEMENT_LIST.click();
```

Comment:
1. the variable name must be meaningful
2. the issue automated was about a problem in the list view. In this TC you didn't check that the view is well displayed.